pgcli/packages/sqlcompletion.py

```diff
@@ -427,8 +430,17 @@ def suggest_based_on_last_token(token, stmt):
 
         return tuple(suggest)
 
-    elif token_v in ('table', 'view', 'function'):
-        # E.g. 'DROP FUNCTION <funcname>', 'ALTER TABLE <tablname>'
+    elif token_v == 'function':
+        schema = stmt.get_identifier_schema()
+        if stmt.get_previous_token(token).value.lower() == 'drop':
```

Comment:
the `if` and `elif` can be combined here, right? 
`if (...).value.lower() in ('drop', 'alter'):`